src/constants/httpStatusCodes.js

```diff
@@ -0,0 +1,10 @@
+const status = {
```

Comment:
```suggestion
const STATUS_CODES = {
```
Constants should be in UPPER_CASE